src/ui/layout/pane.vue

```diff
@@ -78,11 +78,15 @@ export default {
     async mounted() {
         await this.$nextTick();
         // Hide tree and/or inspector pane if specified in URL
-        this.handleHideUrl();
-        this.openmct.router.on('change:params', this.handleHideUrl);
+        if (this.collapsable) {
```

Comment:
Since `collapsable` comes from props, what happens if the prop changes? Do you think we need to add a watcher for the prop and react to any changes?